gateway.go

```diff
@@ -132,5 +134,11 @@ func (s *gatewayResource) configHandler(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	// Make expiration time even/random throughout interval 12-36h
+	// So the avg will be 24h but without spikes of the requests for the key renewal
+	rand.Seed(time.Now().UnixNano())
+	maxAge := 12*3600 + rand.Intn(24*3600)
+	w.Header().Set("Cache-Control", fmt.Sprintf("max-age=%d, private", maxAge))
```

Comment:
```suggestion
	w.Header().Set("Cache-Control", fmt.Sprintf("max-age=%d", maxAge))
```

The configuration is public, so I don't think we need to restrict this configuration from being put in a shared cache.

Reply:
But then we agree that public cache of stalled key is ok. It may be even ok as it reduces big load spikes a bit, but dunno, could you clarify here please?

Reply:
This would seem to be a server issue though. If it decides to publish a new key _before_ the old one expires, then it's breaking any client that has a cached copy of the key. I would suggest that the server MUST NOT do rotate keys faster than the cache TTL.

Reply:
A client that has a cached copy can easily invalidate it over non-cached source at gateway. But if it's cached somewhere outside He just can't re-request successfully.

Reply:
Right, but if the server handles old and new keys, then it wouldn't be a problem. Let's punt this to a followup change! =)